pre-docs/sharing_code_among_tasks.md

```diff
@@ -0,0 +1,41 @@
+
+# Sharing Code Among Tasks
+
+Tasks can use libraries which live alongside the task code, which one or many tasks can access. Tasks can relatively include 
```

Comment:
I don't think this is true yet. This feature only allows multiple tasks to use the same executable